examples/cpp/cpptutorial.md

```diff
@@ -1,488 +1 @@
-protocol buffer object (in this case a `Feature`) until there are no more
-messages: the client needs to check the return value of `Read()` after each
-call. If `true`, the stream is still good and it can continue reading; if
-`false` the message stream has ended. Finally, we call `Finish()` on the stream
-to complete the call and get our RPC status.
-
-The client-side streaming method `RecordRoute` is similar, except there we pass
-the method a context and response object and get back a `ClientWriter`.
-
-```cpp
-    std::unique_ptr<ClientWriter<Point> > writer(
-        stub_->RecordRoute(&context, &stats));
-    for (int i = 0; i < kPoints; i++) {
-      const Feature& f = feature_list_[feature_distribution(generator)];
-      std::cout << "Visiting point "
-                << f.location().latitude()/kCoordFactor_ << ", "
-                << f.location().longitude()/kCoordFactor_ << std::endl;
-      if (!writer->Write(f.location())) {
-        // Broken stream.
-        break;
-      }
-      std::this_thread::sleep_for(std::chrono::milliseconds(
-          delay_distribution(generator)));
-    }
-    writer->WritesDone();
-    Status status = writer->Finish();
-    if (status.IsOk()) {
-      std::cout << "Finished trip with " << stats.point_count() << " points\n"
-                << "Passed " << stats.feature_count() << " features\n"
-                << "Travelled " << stats.distance() << " meters\n"
-                << "It took " << stats.elapsed_time() << " seconds"
-                << std::endl;
-    } else {
-      std::cout << "RecordRoute rpc failed." << std::endl;
-    }
-```
-
-Once we've finished writing our client's requests to the stream using `Write()`,
-we need to call `WritesDone()` on the stream to let gRPC know that we've
-finished writing, then `Finish()` to complete the call and get our RPC status.
-If the status is `OK`, our response object that we initially passed to
-`RecordRoute()` will be populated with the server's response.
-
-Finally, let's look at our bidirectional streaming RPC `RouteChat()`. In this
-case, we just pass a context to the method and get back a `ClientReaderWriter`,
-which we can use to both write and read messages.
-
-```cpp
-std::shared_ptr<ClientReaderWriter<RouteNote, RouteNote> > stream(
-    stub_->RouteChat(&context));
-```
-
-The syntax for reading and writing here is exactly the same as for our
-client-streaming and server-streaming methods. Although each side will always
-get the other's messages in the order they were written, both the client and
-server can read and write in any order — the streams operate completely
-independently.
-
-## Try it out!
-
-Build client and server:
-```shell
-$ make
-```
-Run the server, which will listen on port 50051:
-```shell
-$ ./route_guide_server
-```
-Run the client (in a different terminal):
-```shell
-$ ./route_guide_client
-```
+The content of this page has moved to [gRPC Basics - C++](https://grpc.io/docs/tutorials/basic/cpp).
```

Comment:
Is there ever a situation when I'd need this redirect? What link do I need to visit to still see this page?
If it's unlikely that users will land on this page, let's just remove the file.

Reply:
> If it's unlikely that users will land on this page, let's just remove the file.

Given that it is an `.md` file embedded in a repo, it probably isn't very likely that someone has kept a link. Ok then, I'll delete the file.